mockall/tests/check_expectations_on_object_drop.rs

```diff
@@ -0,0 +1,54 @@
+// vim: tw=80
```

Comment:
You don't need a new test program for this.  The reason that Mockall's tests are split into so many separate test programs is so that most of them will still compile successfully even if one does not.  And when investigating a compile failure, it helps to narrow the search for the problem.
For this PR, you aren't changing mockall's syntax at all.  So I suggest that you delete this file and delete the `expectation_satisfied` and `too_few_calls` methods, which are redundant with tests elsewhere.  Then, move `too_many_calls` into `tests/mock_struct.rs`, in the `checkpoint` module.